sonarqube/content.md

```diff
@@ -55,16 +55,59 @@ By default, the image will use an embedded H2 database that is not suited for pr
 
 Setup a database by following the "Installing the Database" section of https://docs.sonarqube.org/latest/setup/install-server/.
 
-### Use bind-mounted folders
+### Use bind-mounted folders or volumes
 
-The images contain the SonarQube installation at `/opt/sonarqube`. You need to use bind-mounted folders to override selected files or directories :
+-	SonarQube 8.X : Use bind-mounted folders
+-	SonarQube 7.9.X LTS : Use volumes
+
+The images contain the SonarQube installation at `/opt/sonarqube`. You need to bind them to your host to override selected files or directories :
 
 -	`/opt/sonarqube/conf`: configuration files, such as `sonar.properties`
 -	`/opt/sonarqube/data`: data files, such as the embedded H2 database and Elasticsearch indexes
 -	`/opt/sonarqube/logs`: contains SonarQube logs about access, web process, CE process, Elasticsearch logs
 -	`/opt/sonarqube/extensions`: plugins, such as language analyzers
 
-### First installation
+### SonarQube 7.9.X LTS First installation
+
+Follow these steps for your first installation:
+
+1.	Create volumes `sonarqube_conf`, `sonarqube_data`, `sonarqube_logs` and `sonarqube_extensions` and start the image with the following command. This will populate all the folders (copying default plugins, create the elasticsearch data folder, create the sonar.properties configuration file). Watch the logs, and once the container is properly started you can force-exit (ctrl+c) and proceed to the next step.
```

Comment:
```suggestion
1.	Create volumes `sonarqube_conf`, `sonarqube_data`, `sonarqube_logs` and `sonarqube_extensions` and start the image with the following command. This will populate all the volumes (copying default plugins, create the elasticsearch data folder, create the sonar.properties configuration file). Watch the logs, and once the container is properly started you can force-exit (ctrl+c) and proceed to the next step.
```